### PR TITLE
Add ephemeral storage and revise

### PR DESCRIPTION
**Pull request by ritawatson**
Added ephemeral storage for virtual clusters and revamped doc. 

Jira: https://spectrocloud.atlassian.net/browse/PPD-620

[Preview](https://deploy-preview-1029--docs-spectrocloud.netlify.app/devx/resource-quota)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1029#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/04-resource-quota.md [hunk 1]
```diff
@@ -16,36 +16,40 @@ import Tooltip from "shared/components/ui/Tooltip";
...
-1. Login to the Palette console as Tenant Admin, go to Tenant Settings and select `Developer Settings.
+* System-level quota while launching virtual clusters in system-level cluster groups. 
+* Tenant-level quota while launching virtual clusters in their tenant-level cluster groups.
 
+## Manage Developer Quota
 
-2. In the wizard to Manage Developer Settings, there is an option to `Hide system-level cluster groups from tenant users` using the toggle button. You will have to create your cluster groups by hiding the system-level cluster groups.
+To create your cluster groups, you must manage the developer quota in Palette by hiding the system-level cluster groups as follows:
 
+1. Log in to the Palette console.
```
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/04-resource-quota.md:32]:
```suggestion
1. Log in to [Palette](https://console.spectrocloud.com/). 
```

**Review by karl-cardenas-coding — APPROVED** [review]
Minor suggestion, otherwise 👍🏻

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1029#pullrequestreview-1213712951" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1029#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/04-resource-quota.md [hunk 2]
```diff
@@ -16,36 +16,38 @@ import Tooltip from "shared/components/ui/Tooltip";
...
-s
-At the same time, our Enterprise users may offer the capabilities mentioned earlier to their internal developers, i.e., enterprise developers. These enterprise users (in theory) could use the system-level quota offered to every other developer. Still, in all likelihood, the tenant admin will set up different cluster groups(s) and quotas for enterprise developers. You may view this as the enterprise developer's quota offered at the tenant level.
-Enterprise developers may not exceed their system-level quota while launching Palette Virtual Clusters in system-level cluster group(s). In addition, they may not exceed their tenant-level quota while launching Palette Virtual Clusters in their tenant-level cluster groups.
+Palette allocates a quota to users of Palette Cluster Groups as part of our free tier offering. You may launch Palette Virtual Clusters or deploy applications to new or existing virtual clusters. You may use the quota defined at the system level while exploring Palatte SaaS capabilities.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.5-devx/04-resource-quota.md]: fixed typo

File: content/docs/04.5-devx/04-resource-quota.md [hunk 3]
```diff
@@ -16,36 +16,38 @@ import Tooltip from "shared/components/ui/Tooltip";
...
-Enterprise developers may not exceed their system-level quota while launching Palette Virtual Clusters in system-level cluster group(s). In addition, they may not exceed their tenant-level quota while launching Palette Virtual Clusters in their tenant-level cluster groups.
+Palette allocates a quota to users of Palette Cluster Groups as part of our free tier offering. You may launch Palette Virtual Clusters or deploy applications to new or existing virtual clusters. You may use the quota defined at the system level while exploring Palatte SaaS capabilities. 
 
-## Manage the developer Quota
+Palette enterprise users may offer these capabilities to their internal developers. Theoretically, enterprise developers would be able to use the system-level quota offered to other developers, but the tenant admin will most likely set up different cluster groups and quotas for enterprise developers. You can view this as the enterprise developer's quota offered at the tenant level.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.5-devx/04-resource-quota.md:21]: fixed

File: content/docs/04.5-devx/04-resource-quota.md [hunk 4]
```diff
@@ -16,36 +16,40 @@ import Tooltip from "shared/components/ui/Tooltip";
...
-s
-At the same time, our Enterprise users may offer the capabilities mentioned earlier to their internal developers, i.e., enterprise developers. These enterprise users (in theory) could use the system-level quota offered to every other developer. Still, in all likelihood, the tenant admin will set up different cluster groups(s) and quotas for enterprise developers. You may view this as the enterprise developer's quota offered at the tenant level.
-Enterprise developers may not exceed their system-level quota while launching Palette Virtual Clusters in system-level cluster group(s). In addition, they may not exceed their tenant-level quota while launching Palette Virtual Clusters in their tenant-level cluster groups.
+Palette allocates a quota to users of Palette Cluster Groups as part of Spectro Cloud's free tier offering. You can launch Palette Virtual Clusters or deploy applications to new or existing virtual clusters. You may use the quota defined at the system level while exploring Palatte SaaS capabilities.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.5-devx/04-resource-quota.md:19]: fixed typo

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1029#pullrequestreview-1213712951" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1029#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1029#pullrequestreview-1214096478" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**